build.gradle

```diff
@@ -17,9 +17,9 @@ repositories {
 }
 
 dependencies {
-    compile('com.github.Praqma:tracey-broker:master-SNAPSHOT') { changing = true }
+    compile 'com.github.Praqma:tracey-broker:1.0.0'
     compile 'net.sourceforge.argparse4j:argparse4j:0.7.0'
-    testCompile group: 'junit', name: 'junit', version: '4.12'
+    compile 'org.codehaus.groovy:groovy:2.4.7'
```

Comment:
@Andrey9kin  We used groovy lib for file parsing. Missed that after refactoring, thx